cloudevents/formats/avro-format.md

```diff
@@ -148,24 +148,29 @@ For other types (non-binary data without a `datacontenttype` attribute), the
 implementation MUST translate the data value into a representation of the JSON
 value using the union types described for the `data` record.
 
-## 4 Examples
+## 4 Transport
+
+Transports that support content identification MUST use the following designation:
+
+```text
+application/cloudevents+avro
+```
+
+## 5 Examples
 
 The following table shows exemplary mappings:
 
-| CloudEvents | Type   | Exemplary Avro Value                           |
-| ----------- | ------ | ---------------------------------------------- |
-| type        | string | `"com.example.someevent"`                      |
-| specversion | string | `"1.0"`                                        |
-| source      | string | `"/mycontext"`                                 |
-| id          | string | `"7a0dc520-c870-4193c8"`                       |
-| time        | string | `"2019-06-05T23:45:00Z"`                       |
-| dataschema  | string | `"http://registry.com/schema/v1/much.json"`    |
-| contenttype | string | `"application/json"`                           |
-| data        | bytes  | `"{"much":{"wow":"json"}}"`                    |
-|             |        |                                                |
-| dataschema  | string | `"http://registry.com/subjects/ce/versions/1"` |
-| contenttype | string | `"application/avro"`                           |
-| data        | bytes  | `[avro-serialized-bytes]`                      |
+| CloudEvents     | Type   | Exemplary Avro Value                      |
+|-----------------|--------|-------------------------------------------|
+| id              | string | `7a0dc520-c870-4193c8`                    |
+| source          | string | `https://github.com/cloudevents`          |
+| specversion     | string | `1.0`                                     |
+| type            | string | `com.example.object.deleted.v2`           |
+| datacontenttype | string | `application/octet-stream`                |
+| dataschema      | string | `http://registry.com/schema/v1/much.json` |
+| subject         | string | `mynewfile.jpg`                           |
+| time            | long   | `2019-06-05T23:45:00Z`                    |
```

Comment:
Not being an Avro expert this looks odd to have this as a "long" but then show its value as a string.

Reply:
this is correct for avro

Reply:
it's correct but not precise. it's an "int" with the logicalType "date". While we're doing edits here we should probably correct that, maybe as "int (date)"